### ISSUE TITLE
Error when handleActions' map doesn't contain entries

### ISSUE THREAD

**Issue by ruszki** [acdlite/redux-actions#173]
Hi,

When an empty map is passed to handleActions, then Redux is failed because undefined will be returned for undefined state input. This has been already indicated in the discussion of #167 by @geminiyellow. The error message:

> Uncaught Error: Reducer "somekeyinsomestateobject" returned undefined during initialization. If the state passed to the reducer is undefined, you must explicitly return the initial state. The initial state may not be undefined.

The message comes from Redux, but the problem is caused by implementation of redux-actions' handleActions.

Of course there are not many useful use-cases for an empty map. Reducers of such handleActions basically mean that their associated state elements are unmodifiable, which is not recommended, as far as I know. It can be a "feature", but then the documentation should be updated indicating this mandatory condition.

**Comment by yangmillstheory** [comment]
Would you mind submitting a pull request for the missing documentation? Should be a quick change.

**Comment by yangmillstheory** [comment]
Why were you passing in an empty map before?

**Comment by ruszki** [comment]
We started to transform our product to a React/Redux based site from a Thymeleaf based one. Now, the server generates an object which is used as an initial state by Redux. So some state elements are not yet under the control of Redux, and thus, they are read-only. We used handleActions({}, defaultState) as a dummy reducer. It was chosen to maintain consistency with other reducers, so it was just a matter of taste.
